### PR TITLE
Merge to main

### PR DESCRIPTION
**Pull request by MaxWardle**
please see the changes here: https://dev.developer.gov.bc.ca/docs/default/component/public-cloud-techdocs

### CODE REVIEW

**Review by wrnu — APPROVED** [review]

File: docs/aws-billing-and-cost-management-dashboard-via-quicksight.md [hunk 1]
```diff
@@ -2,14 +2,16 @@
 Last updated: **March 28, 2024**
```
Comment (Pilargit12) [inline docs/aws-billing-and-cost-management-dashboard-via-quicksight.md:2]:
```suggestion
Last updated: **August 26, 2024**
```

File: docs/iam-user-service.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ To create an IAM user:
 
     A Lambda function will trigger to create the IAM user, generate an access key, and store it in the SSM Parameter Store. This function also runs hourly to rotate keys as needed and ensure DynamoDB table entries align with actual IAM account users, removing any discrepancies.
 
-      ![iam-user](images/iam-user-service/iam-users.png)
+    ![iam-user](images/iam-user-service/iam-users.png)
```
Comment (Pilargit12) [inline docs/iam-user-service.md:23]: Hi @MaxWardle  can you please create a folder called images, upload the images there and then the URL should be: 

[Image example](../../images/imagename.png)

**Review by Pilargit12 — COMMENTED** [review]
Please have a look at the images comment before proceeding.

File: docs/iam-user-service.md (hunk 2 above)
Comment (MaxWardle) [reply docs/iam-user-service.md:23]: Hey @Pilargit12, this folder exists [here](https://github.com/bcgov/public-cloud-techdocs/tree/dev/docs/images) already, this change was a formatting issue. Markdown doesn't like that space I removed, but the image should render now.

**Review by Pilargit12 — APPROVED** [review]

**Pull request merged**